rollbar-java/src/main/java/com/rollbar/notifier/filter/Filter.java

```diff
@@ -21,6 +21,27 @@
    */
```

Comment:
I think we can mark this method as deprecated.

Reply:
I was trying to change this interface so that this change would not be a breaking change by deprecating this and providing a default implementation of the method below. But it looks like default interface implementations are not supported before Java 8, so the only thing I can see to do is to make a semver major breaking change.